Makefile

```diff
@@ -153,3 +153,14 @@ run-dev-server: sql/target/host/dev/skdb build/init.sql
 build/index.html: sql/js/index.html
 	mkdir -p build
 	cp $^ $@
+
+.PHONY: check-vite
+check-vite: npm
+	rm -rf build/vitejs
+	git clone https://github.com/SkipLabs/skdb_vite_react_starter.git build/vitejs
+	cd build/vitejs && npm install;
+	rm -r build/vitejs/node_modules/skdb
+	cp -r build/package/skdb build/vitejs/node_modules/
+	cd build/vitejs && npm run build
```

Comment:
Yes! It might also be worth try to test `npm run dev`. I found often enough that one could be working while the other was broken. But this might be awkward to test actually as you'll need to load the page. :/